stories/features/rewards/concepts.tsx

```diff
@@ -466,61 +466,3 @@ storiesOf('Feature Components/Rewards/Concepts/Mobile', module)
     />
   ))
   .add('Settings', () => <SettingsMobile />)
-  .add('Site Banner', withState({ donationAmounts, currentAmount: '5.0', showBanner: true }, (store) => {
```

Comment:
why was this one removed?

Reply:
It is only for mobile, which was removed. (notice the `isMobile={true}` prop)

Reply:
Can confirm, this is fine to remove 👍